app/scripts/directives/bindService.js

```diff
@@ -151,89 +177,36 @@ function BindService($filter,
         }
         sortServiceInstances();
       });
+      ctrl.appToBind = ctrl.target;
     }
     // TODO: handle not having any service instances when binding app to service
-    ctrl.gotoStep(ctrl.steps[0]);
-  };
-
-  var humanizeKind = $filter('humanizeKind');
-  ctrl.groupByKind = function(object) {
-    return humanizeKind(object.kind);
   };
 
```

Comment:
nit: I tend to prefer `$onInit` at the beginning and `$onDestroy` at the end to make it easy/consistent to find the setup/tear down code. Not a big issue, just a pattern preference.